tools/DiagramNSX/NsxObjectCapture.ps1

```diff
@@ -266,6 +268,11 @@ Get-NsxFirewallSection | Get-NsxFirewallRule -connection $connection | % {
     $DfwRuleHash.Add($_.id, $_.outerxml)
 }
 
+write-host "  Getting Security Policies"
+Get-NsxSecurityPolicy | -connection $connection | % {
```

Comment:
The pipe before the -connection looks like it shouldn't be there... Did you test?